src/autoscaler/eventgenerator/config/config_test.go

```diff
@@ -155,7 +162,7 @@ db:
   policy_db_url: postgres://postgres:password@localhost/autoscaler?sslmode=disable
   app_metrics_db_url: postgres://postgres:password@localhost/autoscaler?sslmode=disable
 aggregator: 
-  aggregator_execute_interval: NOT-INTEGER-VALUE
+  aggregator_execute_interval: 5k
```

Comment:
why change the value? 5k is not as clear as NOT-INTEGER-VALUE.

Reply:
it is used to check whether it can parse a time duration, not an integer. The value can be 5s/5m etc, they are not integer but valid time duration